### PR TITLE
New game form route 

### PR DESCRIPTION
**Pull request by Vithop**
Created a  new page to provide a form to initialize the game

### CODE REVIEW

**Comment by Vithop** [comment]
I should make a git hook to reformat commits

**Comment by Vithop** [comment]
image
image

File: svelte/src/routes/NewGame/+page.server.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  playAgainstAI: boolean;
+}
+
+export const actions = {
+  default: async ({ cookies, request }) => {
+    console.debug(cookies, request);
+
+    const data = await request.formData();
+    console.debug(data);
+    const gameData: NewTicTacToeGameFormData = {
```
[the remark this answers is not in the document]
Comment (Vithop) [reply svelte/src/routes/NewGame/+page.server.ts:15]: Will update to TicTacToeSettings

File: svelte/src/routes/NewGame/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import "papercss/dist/paper.min.css";
+  import { Button } from "spaper";
+</script>
+
+<div class="settings paper row flex-center">
+  <form method="POST">
```
[the remark this answers is not in the document]
Comment (Vithop) [reply svelte/src/routes/NewGame/+page.svelte:7]: There's no issue here

**Review by Mathuran — APPROVED** [review]

**Pull request merged**